tutorials/noisepy_datastore.ipynb

```diff
@@ -0,0 +1,208 @@
+   },
+   "source": [
+    "#  Introduction to NoisePy DataStore Class\n",
+    "\n",
+    "Noisepy is a python software package to process ambient seismic noise cross correlations. \n",
+    "\n",
+    "**Publication about this software**:\n",
+    "Chengxin Jiang, Marine A. Denolle; NoisePy: A New High‐Performance Python Tool for Ambient‐Noise Seismology. Seismological Research Letters 2020; 91 (3): 1853–1866. doi: https://doi.org/10.1785/0220190364"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Uncomment and run this line if the environment doesn't have noisepy already installed:\n",
+    "# ! pip install noisepy-seis "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "__Warning__: NoisePy uses ```obspy``` as a core Python module to manipulate seismic data. Restart the runtime now for proper installation of ```obspy``` on Colab."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "FaKuajVCOo2r"
+   },
+   "source": [
+    "This tutorial should be ran after installing the noisepy package. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "vceZgD83PnNc"
+   },
+   "outputs": [],
+   "source": [
+    "from noisepy.seis import cross_correlate, stack, plotting_modules, __version__       # noisepy core functions\n",
+    "from noisepy.seis.scedc_s3store import SCEDCS3DataStore, channel_filter # Object to query SCEDC data from on S3\n",
+    "from noisepy.seis.channelcatalog import XMLStationChannelCatalog        # Required stationXML handling object\n",
+    "from datetime import datetime\n",
+    "from datetimerange import DateTimeRange\n",
+    "\n",
+    "print(f\"Using NoisePy version {__version__}\")\n",
+    "\n",
+    "S3_STORAGE_OPTIONS = {\"s3\": {\"anon\": True}}"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "yojR0Z3ALm6K"
+   },
+   "outputs": [],
+   "source": [
+    "# timeframe for analysis\n",
+    "start = datetime(2002, 1, 2)\n",
+    "end = datetime(2002, 1, 4)\n",
+    "time_range = DateTimeRange(start, end)\n",
+    "print(time_range)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## DataStore"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Data Store is a set of classes that \n",
+    "Various types of data store are necessary because how reseachers store their data can be dramatically different w.r.t. formats (mSEED, SAC, SEG-Y), file system (local, S3), naming conventions. It is not a good idea forcing the users to follow a specific type of data storage. A Data Store class can be the front-end of the real back-end data storage, and return data through `read_data` function. It allows users to customize based on how they store the data, and leaving the rest of the workflow untouched. \n",
```

Comment:
suggested changes in the text: " A noisepy DataStore is a set of classes to accommodate the various types of data store that are necessary because how reseachers store their data, which can be dramatically different w.r.t. formats (mSEED, SAC, SEG-Y), file system (local, S3), and naming conventions. Our noisepy team does not impose a definite data structure, but instead suggest to wrap the data storage structure into a python class. A Data Store class can be the front-end of the real back-end data storage, and return data through read_data function. It allows users to customize based on how they store the data, and leaving the rest of the workflow untouched."